src/templates/stake-page.js

```diff
@@ -18,77 +20,58 @@ export const StakePageTemplate = ({
 }) => {
   console.log("tech:", images)
   return (
-    <div className="main-content">
+    <div className="stake-content">
       <PageSection id={sections.stake.HOME}>
         <Row>
-          <Col xs={12}>
-            <h1 dangerouslySetInnerHTML={{ __html: hero.title }} />
+          <Col className="test-stake" xs={12} md={7}>
```

Comment:
@astanovferuz, seems like you missed to remove the class `test-stake` for your debug purpose.